src/main/java/org/jitsi/jigasi/sounds/SoundNotificationManager.java

```diff
@@ -624,7 +624,7 @@ public void notifyChatRoomMemberJoined(ChatRoomMember member)
     public void notifyChatRoomMemberLeft(ChatRoomMember member)
     {
         // if this is the sip hanging up (stopping) skip playing
-        if (gatewaySession.getJvbConference().isStarted()
+        if (gatewaySession.getJvbConference() != null && gatewaySession.getJvbConference().isStarted()
```

Comment:
AbstractGatewaySession.jvbConference can change to null, so this can still produce an NPE (granted, rarely)

Reply:
Updated.